LaLauncher/src/main/res/anim/slide_down.xml

```diff
@@ -2,7 +2,7 @@
 <set xmlns:android="http://schemas.android.com/apk/res/android">
 
     <translate
-            android:duration="400"
```

Comment:
Slides are already removed on the epaper branch from the code, so I'suggest to remove this xml completely